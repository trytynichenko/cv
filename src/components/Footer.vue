<template>
  <footer class="footer block">
    <p>Made with <img src="../assets/images/fire.png"/> by {{ author }} &copy; {{ year }}</p>
    <p></p>
  </footer>
</template>

<script>
export default {
  name: 'Footer',
  data: () => ({
    author: 'Leonid Trytynichenko',
  }),
  computed: {
    year: () => new Date().getFullYear(),
  },
};
</script>

<style scoped lang="scss">
  @import '../assets/styles/components/footer';
</style>
