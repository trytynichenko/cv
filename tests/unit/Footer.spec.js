import { shallow } from '@vue/test-utils';
import Footer from '@/components/Footer.vue';

describe('Footer.vue', () => {
  const data = {
    author: 'Bob',
  };

  const wrapper = shallow(Footer, {
    data,
  });

  it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
