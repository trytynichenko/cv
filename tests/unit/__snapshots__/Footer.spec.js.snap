// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue matches snapshot 1`] = `
<footer class="footer block">
    <p>Made with
        <img src="../assets/images/fire.png"> by Bob Â© 2018</p>
    <p></p>
</footer>
`;
